cpp/comms/std/src/cuML_std_comms_impl.cpp

```diff
@@ -220,7 +220,13 @@ cumlStdCommunicator_impl::cumlStdCommunicator_impl(
 
 cumlStdCommunicator_impl::cumlStdCommunicator_impl(ncclComm_t comm, int size,
                                                    int rank, bool verbose)
-  : _nccl_comm(comm), _size(size), _rank(rank), _verbose(verbose) {
+  : _nccl_comm(comm),
+    _size(size),
+    _rank(rank),
+    _verbose(verbose),
+    _ucp_worker(nullptr),
+    _ucp_handle(nullptr),
+    _ucp_eps(nullptr) {
```

Comment:
when `WITH_UCX` is undefined, all these member variables will also be not defined. So, this should cause a compilation failure, no?

Reply:
Good catch. This was initially for debugging but we lack automated testing for `WITH_UCX=OFF`. Do you have any thoughts on how we might best test this path?